README.md

```diff
@@ -1 +1,36 @@
-Snowplow Data Models Repo
+![snowplow-logo](snowplow_logo.jpg)
+
+Snowplow is a scalable open-source platform for rich, high quality, low-latency data collection. It is designed to collect high quality, complete behavioural data for enterprise business.
+
+# Snowplow Pipeline Overview
+
+![snowplow-pipeline](snowplow_pipeline.png)
```

Comment:
Can we fix this image to not have the autocorrect squiggles?

Reply:
Ah my bad. Will fix.